snap-css-web/src/views/Home.vue

```diff
@@ -69,22 +71,22 @@
 
 <script lang="ts">
 /* eslint-disable @typescript-eslint/no-explicit-any */
-import { Options, Vue } from "vue-class-component";
-import HelloWorld from "@/components/HelloWorld.vue"; // @ is an alias to /src
-import SnapCss from "snappy-css";
-import Optimizer from "snappy-css/lib/optimize/optimizer";
-import Editor from "@/components/Editor.vue";
+import { Options, Vue } from 'vue-class-component';
```

Comment:
Is there a rule stating the way to write single quotes instead of double?
maybe better to specify that if it is not configured that way (if eslint exists that is)